uiowa/uiowa
# Refactor of iisc projects page

Resolves https://github.com/uiowa/uiowa/issues/5714. 

# How to test

- `blt frontend`
- `ddev blt ds --site=iisc.uiowa.edu`
- Go to `/projects` and verify that search appearance has changed and now has an advanced filters section
- Verify that advance filters auto submit while the title field does not auto submit
- Verify that partners appear in card teaser on `/projects` results page
- Go to https://iisc.prod.drupal.uiowa.edu/partners/jackson-county and verify that results are paged.
